glib/src/gobject/binding_group.rs

```diff
@@ -120,7 +120,10 @@ impl<'a> BindingGroupBuilder<'a> {
                 None => false,
                 Some(res) => {
                     assert!(
-                        res.type_().is_a(transform_data.3.value_type()),
+                        crate::Value::type_transformable(
+                            res.type_(),
+                            transform_data.3.value_type()
+                        ),
```

Comment:
Why this change?

Reply:
I think the check is more correct, as it should accept something that is transformable to the actual type and not exactly the same one (it was what it it was suggested for the other closure too), but it needs a test

Reply:
Yes please. Is something actually going to do that transform at some point ;)

Also this probably means that `GBinding` bindings need to be updated the same way?

Reply:
Also unclear if we want to allow that. This means that you can pass a `0u32` and it would become a `false`, for example. For properties/signals we disallow automatic conversions as that seems out of place in Rust.

Reply:
Let's leave it as `type_().is_a`, that should disallow numeric conversions but still allow for object/interface types to be passed without casting them

Reply:
@A6GibKm This is still pending

Reply:
Ok leaved as it was.